.gitignore

```diff
@@ -3,3 +3,4 @@ gitignore/
 venv/
 *.log
 certbot.out
+backend/curriculum_tracking/tests/for_testing_purposes.py
```

Comment:
Why not just delete the file? 

I'm worried that out gitignore will become very noisy if we just put lots of random stuff in there.